module/__fortran_builtins.f90

```diff
@@ -0,0 +1,15 @@
+!===-- module/__fortran_builtins.f90 ---------------------------------------===!
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+!
+!------------------------------------------------------------------------------!
+
+! These naming shenanigans prevent names from Fortran intrinsic modules
+! from being usable on INTRINSIC statements, and force the program
+! to USE the standard intrinsic modules in order to access the
+! standard names of the procedures.
+module __Fortran_builtins
```

Comment:
I'm not sure I understand the comment. Is there anything that prevents user code from using `__Fortran_builtins` and calling `__builtin_c_f_pointer`? Should there be?

Reply:
No, there isn't, and I'm not sure that there needs to be one -- it should be obvious that the intrinsics in this module are implementation artifacts.